sdk/src/main/scala/com/horizen/validation/ConsensusValidator.scala

```diff
@@ -0,0 +1,64 @@
+package com.horizen.validation
+import java.time.Instant
+
+import com.horizen.SidechainHistory
+import com.horizen.block.SidechainBlock
+import com.horizen.consensus.{NonceConsensusEpochInfo, _}
+import scorex.util.ScorexLogging
+
+import scala.util.Try
+
+class ConsensusValidator extends HistoryBlockValidator with ScorexLogging {
+  override def validate(block: SidechainBlock, history: SidechainHistory): Try[Unit] = Try {
+    if (history.isNotGenesisBlock(block.id)) {
```

Comment:
the only usage of `isNotGenesisBlock`. 
We can check if `isGenesisBlock` else ...
and remove `isNotGenesisBlock` method at all.